tuf/api/metadata.py

```diff
@@ -25,7 +26,7 @@
 from securesystemslib.storage import FilesystemBackend, StorageBackendInterface
 from securesystemslib.util import persist_temp_file
 
-from tuf import exceptions
+from tuf import SPECIFICATION_VERSION, exceptions
```

Comment:
I've mentioned already that I think this is not correct -- we have two separate implementations in our code base and I don't see why this variable would the one thing that they share after sharing almost nothing else.

Using tuf.SPECIFICATION_VERSION also means you end up writing code you don't need: `SPECIFICATION_VERSION.split(".")` every time a signed is initialized. Instead you could define the spec version in metadata.py as a tuple of strings which you an then directly compare (`spec_list[0] != SPEC_VERSION[0]`)

Reply:
I agree with you, the only reason I reused `tuf. SPECIFICATION_VERSION` is that in the python-tuf slack channel when I brought that question I got a mixed answer.
There were opinions we shouldn't bump the `tuf. SPECIFICATION_VERSION` to `1.0.19` for example, because we haven't confirmed we support it. See the channel for details.

I will add a constant(s) in `tuf.api` and we will discuss should we bump it later.